### ISSUE TITLE
Multiple aliases?

### ISSUE THREAD

**Issue by lorengordon** [PoweredLocal/mailtumble#1]
Hi, this looks really interesting, nice work! I was skimming the code but wasn't quite able to determine, does this currently support resolving one inbound email address to multiple recipients? I.e. inbound email with a single recipient `foo@example.com` will send the message to both `bar@bardomain.com` and `baz@bazdomain.com`?

**Comment by dusterio** [comment]
It doesn't but it's trivial to add this - since AWS SES is used to send out emails, the recipient email is always an array, it's just that at the moment it only contains one address.

All it takes is to store an array of emails in the alias database/table, this will probably affect only one function in the code

**Comment by lorengordon** [comment]
Yeah it appeared to be this general area to me, along with managing the dynamo table perhaps a bit differently:

https://github.com/PoweredLocal/mailtumble/blob/3fa22e3d28384f06672982352bf945fc8e74fbcb/receive.js#L80-L91

(Javascript is still a bit like black magic to me, so I may be way off)
